frontend/src/layout/navigation-3000/navbarItems.tsx

```diff
@@ -27,18 +30,46 @@ export const NAVBAR_ITEMS: NavbarItem[][] = [
             icon: <IconCottage />, // This is in a separate section as it uniquely is a direct URL
             pointer: urls.projectHomepage(),
         },
-    ],
-    [
         {
             identifier: Scene.Dashboards,
             label: 'Dashboards',
             icon: <IconGauge />,
             pointer: dashboardsSidebarLogic,
         },
+        {
+            identifier: Scene.Cohorts,
+            label: 'Cohorts',
+            icon: <IconCohort />,
+            pointer: cohortsSidebarLogic,
+        },
+        {
+            identifier: Scene.Persons,
+            label: 'Persons and groups',
+            icon: <IconPerson />,
+            pointer: personsAndGroupsSidebarLogic,
+        },
+        {
+            identifier: Scene.Events,
+            label: 'Events',
+            icon: <IconLive />,
+        },
+        {
+            identifier: Scene.DataManagement,
+            label: 'Data management',
+            icon: <IconUnverifiedEvent />,
+        },
+        {
+            identifier: Scene.Annotations,
+            label: 'Annotations',
+            icon: <IconComment />,
+        },
+    ],
+    [
         {
             identifier: Scene.SavedInsights,
-            label: 'Insights',
+            label: 'Analytics',
```

Comment:
That name is way more understandable. We should also change the name throughout the rest of the UI at some point.